jwst/scripts/schemadoc.py

```diff
@@ -30,16 +32,25 @@
 # DAMAGE.
 
 import argparse
-import sys
+import logging
+from pathlib import Path
 
 from stdatamodels.schema import build_docstring
 
 
-def get_docstrings(template, model_names, all=False):
-    # Get the docstring for every model class
+# Set logger to only print to screen
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+stream_handle = logging.StreamHandler()
+stream_handle.setLevel(logging.INFO)
+logger.addHandler(stream_handle)
+
+
+def get_docstrings(template, model_names, all_models=False):
+    """Get the docstring for every model class."""
     from stdatamodels.jwst.datamodels import _defined_models as defined_models
```

Comment:
I know you didn't write this, but why is this import inside the function?

Reply:
yep, I didn't like this either so I moved it up with the other imports.